Document management address (VIP) requirements (#537)
harvester/docs: docs/install/management-address.md

{"kube-vip.io/hwaddr":"02:00:00:09:7f:3f","kube-vip.io/requestedIP":"10.84.102.31"}
```

> ## Finding which node the VIP is on
> 
> Using `kubectl` (either on your own machine with the Harvester kubeconfig, or SSH'd into any Harvester management node as root):

Reviewer: ```suggestion
## Identifying the Node the VIP Is Assigned To

You can use kubectl (either on your local machine with the Harvester kubeconfig file, or when using SSH to connect to any Harvester management node as a root user).
```

```console
kubectl -n kube-system get svc ingress-expose -o jsonpath='{.metadata.annotations.kube-vip\.io/vipHost}'
```

Example output:
```console
harvester-xzj76
```

> Alternatively, you can SSH into each Harvester management node and run:

Reviewer: ```suggestion
Alternatively, you can use SSH to connect to each Harvester management node and then run the command `ip address show mgmt-br`.

Example:
```

```console
harvester-xzj76:~ # ip address show mgmt-br
4: mgmt-br: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 8c:dc:d4:b5:f0:fd brd ff:ff:ff:ff:ff:ff
    inet 172.19.108.45/21 brd 172.19.111.255 scope global mgmt-br
       valid_lft forever preferred_lft forever
    inet 172.19.108.34/32 scope global mgmt-br
       valid_lft forever preferred_lft forever
```

> When you see both the VIP and the node address in the output of that command, then that means that the node you are running on is the one with the VIP attached to it.

Reviewer: ```suggestion
When the output includes both the VIP and the node address, the VIP is assigned to the node that you are connected to.
```

{"kube-vip.io/hwaddr":"02:00:00:09:7f:3f","kube-vip.io/requestedIP":"10.84.102.31"}
```

## Finding which node the VIP is on

Using `kubectl` (either on your own machine with the Harvester kubeconfig, or SSH'd into any Harvester management node as root):

```console
kubectl get pod -n harvester-system -o custom-columns='NAME:.metadata.name,NODE:.spec.nodeName' | grep -E '^kube-vip|^NAME'
```

Example output:
```console
NAME                                                    NODE
kube-vip-hxzfk                                          harvester-xzj76

> ```

Reviewer 2: This part might need to be more precise. The `kube-vip` Pods run on control-plane nodes, so if there are three Harvester management nodes, three Pods will be listed in the command output, and people still don't know which node the VIP address is on.

I'd suggest we could use the following instead:

```shell
kubectl -n kube-system get svc ingress-expose -o jsonpath='{.metadata.annotations.kube-vip\.io/vipHost}'
```
